File: Modul4/C/PRAK404-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    while (1) {
        int pilihan;
        float nilaiPertama, nilaiKedua, hasil;
        printf("Pilih Program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilihan);
        if (pilihan == 5) {
            char namaAnda[] = "Muhammad Azrianzan";
            printf("Terimakasih telah menggunakan kalkulator %s\n", namaAnda);
            printf("\n");
            break;
        }
        else if (pilihan == 1) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilaiPertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilaiKedua);
            hasil = nilaiPertama + nilaiKedua;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, hasil);
            printf("\n");
        }
        else if (pilihan == 2) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilaiPertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilaiKedua);
            hasil = nilaiPertama - nilaiKedua;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, hasil);
            printf("\n");
        }
        else if (pilihan == 3) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilaiPertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilaiKedua);
            hasil = nilaiPertama * nilaiKedua;
            printf("Hasil pengkalian antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, hasil);
            printf("\n");
        }
        else if (pilihan == 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilaiPertama);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilaiKedua);
            hasil = nilaiPertama / nilaiKedua;
            printf("Hasil penmbagian antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, hasil);
            printf("\n");
        }
        else {
            printf("Input anda salah, silahkan coba lagi\n");
            printf("\n");
        }   
    }
    return 0;
}

File: Modul4/Python/PRAK404-2310817210003-MuhammadAzrianzan.py
while True:
    print("Pilih Program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan = int(input("Masukkan Pilihan : "))
    if pilihan == 5:
        NAMAANDA = "Muhammad Azrianzan"
        print(f"Terimakasih, telah menggunakan kalkulator {NAMAANDA}")
        print("")
        break
    elif pilihan == 1:
        nilaiPertama = int(input("Masukkan nilai pertama : "))
        nilaiPertama = float(nilaiPertama)
        nilaiKedua = int(input("Masukkan nilai kedua : "))
        nilaiKedua = float(nilaiKedua)
        hasil = nilaiPertama + nilaiKedua
        print(f"Hasil penjumlahan antara {nilaiPertama:.2f} dengan {nilaiKedua:.2f} adalah {hasil:.2f}")
        print("")
    elif pilihan == 2:
        nilaiPertama = int(input("Masukkan nilai pertama : "))
        nilaiPertama = float(nilaiPertama)
        nilaiKedua = int(input("Masukkan nilai kedua : "))
        nilaiKedua = float(nilaiKedua)
        hasil = nilaiPertama - nilaiKedua
        print(f"Hasil pengurangan antara {nilaiPertama:.2f} dengan {nilaiKedua:.2f} adalah {hasil:.2f}")
        print("")
    elif pilihan == 3:
        nilaiPertama = int(input("Masukkan nilai pertama : "))
        nilaiPertama = float(nilaiPertama)
        nilaiKedua = int(input("Masukkan nilai kedua : "))
        nilaiKedua = float(nilaiKedua)
        hasil = nilaiPertama * nilaiKedua
        print(f"Hasil pengalian antara {nilaiPertama:.2f} dengan {nilaiKedua:.2f} adalah {hasil:.2f}")
        print("")
    elif pilihan == 4:
        nilaiPertama = int(input("Masukkan nilai pertama : "))
        nilaiPertama = float(nilaiPertama)
        nilaiKedua = int(input("Masukkan nilai kedua : "))
        nilaiKedua = float(nilaiKedua)
        hasil = nilaiPertama / nilaiKedua
        print(f"Hasil pengurangan antara {nilaiPertama:.2f} dengan {nilaiKedua:.2f} adalah {hasil:.2f}")
        print("")
    else:
        print("Input anda salah, silahkan coba lagi")
        print("")